src/yaml/next/serialization.cr

```diff
@@ -0,0 +1,199 @@
+module YAML
+  annotation Field
+  end
+
+  module Serializable
+    annotation Options
+    end
+
+    macro included
+      def self.new(ctx : YAML::ParseContext, node : YAML::Nodes::Node)
```

Comment:
Why not remove this like I commented before?!

Reply:
Hack ;-)